rockcraft/application.py

```diff
@@ -67,3 +69,14 @@ def _enable_craft_parts_features(self) -> None:
         # enable the craft-parts Features that we use here, right before
         # loading the project and validating its parts.
         Features(enable_overlay=True)
+
+    def _get_build_base(self) -> str | None:
+        """Get the project's build-base, if the project file exists."""
+        project_file = self.project_dir / "rockcraft.yaml"
```

Comment:
You should be able to use the `ProjectService`'s `get_raw` here.

Since we'll be extending this functionality to other crafts, we should probably add a `get_build_base` to the `ProjectService` that does this for us.

Reply:
oh good call. I thought that the loading of plugins happened early enough that the project service hadn't even been 'early configured', but I was wrong! Updated